exporter/signalfxexporter/translation/constants.go

```diff
@@ -520,6 +323,7 @@ translation_rules:
   mapping:
     disk.ops: disk_ops.total
 
+## remove after MEAT-2298
```

Comment:
Could you also add a similar TODO for `interface` dimension renaming on network metrics?

Reply:
done